ACPCampaignClassic.podspec

```diff
@@ -1,27 +1,60 @@
 Pod::Spec.new do |s|
   s.name         = "ACPCampaignClassic"
-  s.version      = "1.1.0"
-  s.summary      = "Campaign Classic framework for Adobe Experience Platform SDK. Written and Supported by Adobe."
+  s.version      = "2.0.0"
+  s.summary      = "Campaign Classic framework for Adobe Experience Cloud SDK. Written and maintained by Adobe."
```

Comment:
Should be:

`Campaign Classic library for Adobe Experience Cloud SDK. Written and maintained by Adobe.`